packages/button/test/button.test.ts

```diff
@@ -219,6 +219,24 @@ describe('Button', () => {
             expect(el).to.not.be.undefined;
             expect(el.textContent).to.include('With Target');
         });
+        it('allows link click', async () => {
```

Comment:
This test also passes without the changes on this PR. Is it expected?

Reply:
@rubencarvalho refactored to use a `SendMouse` which will also catch a voiceover click